### DISCUSSION TITLE
Modify the data on a VPS

### DISCUSSION THREAD

**Discussion by DavideWiest** [vrtmrz/obsidian-livesync#356] [Q&A] [answered]
Is there a way to sync a vault to and from a VPS? Or: Can you modify the couch db entries without breaking the synchronization? 
My intention is to run programs on my vps, mostly with LLMs, to edit my vault.

**Comment by vrtmrz** [comment, marked as answer]
Thank you for asking me! And, apologies for being absent!
Yes, we can use [LiveSync-Bridge](https://github.com/vrtmrz/livesync-bridge) to synchronise files headless.
And, If you are familiar with TypeScript, there is `DirectFileManupulator` in [livesync-commonlib](https://github.com/vrtmrz/livesync-commonlib/blob/main/src/DirectFileManipulator.ts) (you know that we lacking the documentation).

I hope that this could help you!
